samples/boards/hexiwear/haptic/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _blinky-sample:
```

Comment:
looks like another case of copying a documentation example and not updating it for the "haptic" sample